packages/common/src/index.ts

```diff
@@ -79,17 +93,19 @@ export default class Common {
     })
   }
 
-  private static _getChainParams(chain: string | number): Chain {
+  private static _getChainParams(chain: string | number, customChains?: Chain[]): Chain {
+    const initializedChains: any = _getInitializedChains(customChains)
     if (typeof chain === 'number') {
-      if (chainParams['names'][chain]) {
-        return chainParams[chainParams['names'][chain]]
+      if (initializedChains['names'][chain]) {
+        const name: string = initializedChains['names'][chain]
+        return initializedChains[name]
       }
 
       throw new Error(`Chain with ID ${chain} not supported`)
     }
 
-    if (chainParams[chain]) {
-      return chainParams[chain]
+    if (initializedChains[chain]) {
+      return initializedChains[chain]
```

Comment:
Ok, so I can only get a custom chain by name, not by ID, right?

Reply:
No, that's the same logic as before here - `chainParams` -> `initializedChains` is just a renaming. The code for returning from a number is some lines above and still works for baked-in as well as custom chains. In fact baked-in chains and custom chains are treated exactly the same throughout the whole library I would assume (if I am not missing something).

Reply:
Yup you are right, I misread this code.